rules/rules-reviewed/eap8/eap7/picketlink.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="picketlink"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Java EE applications that need to remove their picketlink configuration
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[8,9)"/>
+    </metadata>
+    <rules>
+        <rule id="picketlink-00010">
+            <when>
+                <or>
+                    <file filename="{*}WEB-INF/{*}picketlink.xml"/>
```

Comment:
This particular line does not work. I've verified it this way:
- Keep only the condition `<file filename="{*}WEB-INF/{*}picketlink.xml"/>` and remove the other conditions so the final value is:

```
                <or>
                     <file filename="{*}WEB-INF/{*}picketlink.xml"/>
                </or>
```

- Execute the test `mvn test -DrunTestsMatching=picketlink.windup.test.xml`
- The test should fail, which is expected, because the number of hints expected don't match. Just ignore it.
- Open the generated report at `rules/target/WindupRulesTests/windupgraph_RANDON_STRING/reports`
- In your browser navigate to the page `Issues` and you will see that the file `picketlink.xml` did not generated any issue despite the fact that that file is located at the folder `WEB-INF` as the rule expects